pytests/tests/test_clean.py

```diff
@@ -0,0 +1,41 @@
+#            (c) 2019 VMware Inc.,
+#
+#   Author: Siddharth Chandrasekaran <csiddharth@vmware.com>
+
+import pytest
+
+@pytest.fixture(scope='module', autouse=True)
+def setup_test_cache(utils):
+    yield
+    teardown_test_cache(utils)
+
+def teardown_test_cache(utils):
+    pass
+
+def test_clean_no_args(utils):
+    ret = utils.run([ 'tdnf', 'clean' ])
+    assert (ret['retval'] == 135)
```

Comment:
Can we have variable (tells what is the return code expected) assigned with the return values instead of hard coded numbers similar to this https://github.com/vmware/tdnf/blob/036047110bc8e27a5e12bc05d761f83dbfaf93b2/include/tdnfclierror.h#L13 ?

Reply:
Yes, that's ideally what we should do. Python's subprocess module returns only the lower 8 bits of the process return code (as in a shell); therefore there is no 1-1 mapping of those macros to actual return codes.

I'm yet to figure out a way to get all 32-bits so we can have this. Let's put this as an action item for the future.